app/javascript/dashboard/store/modules/specs/macros/actions.spec.js

```diff
@@ -0,0 +1,162 @@
+import axios from 'axios';
+import { actions } from '../../macros';
```

Comment:
@fayazara Can you confirm if this test is executing properly?

Reply:
Fixed this with the correct method